"[ERROR]   DataRetrieverApiIT.testRetrieveMyDataCollections:192 expected: <Root> but was: <dvc9c63c9f>",

We're seeing the following error...

```
"[ERROR]   DataRetrieverApiIT.testRetrieveMyDataCollections:192 expected: <Root> but was: <dvc9c63c9f>",
```

... in the last three runs on the develop branch on Jenkins:

- https://jenkins.dataverse.org/job/IQSS-dataverse-develop/2496/consoleFull
- https://jenkins.dataverse.org/job/IQSS-dataverse-develop/2495/consoleFull
- https://jenkins.dataverse.org/job/IQSS-dataverse-develop/2494/consoleFull

The last PR to be merged before these failures was this one:

- #12425

I made that pull request but I don't think it's actually related.

--- a/src/test/java/edu/harvard/iq/dataverse/api/DataRetrieverApiIT.java
+++ b/src/test/java/edu/harvard/iq/dataverse/api/DataRetrieverApiIT.java
@@ -187,10 +187,14 @@ public class DataRetrieverApiIT {
         // The count should show the list size to be only Root Dataverse count
         items = retrieveMyCollectionListResponse.getBody().jsonPath().getList("data.items");
         assertEquals(rootCount, items.size());
-        // Verify the name and alias of the Root Dataverse. We don't know the id so just make sure it's in the response
-        assertNotNull(items.get(0).get("id"));
-        assertEquals("Root", items.get(0).get("name"));
-        assertEquals("root", items.get(0).get("alias"));
+        // Verify the alias of the Root Dataverse is in the response
+        boolean found = false;
+        for  (int i = 0; i < items.size(); i++) {
+            if ("root".equalsIgnoreCase(items.get(i).get("alias"))) {
+                found = true;
+            }
+        }
+        assertTrue(found, "Root dataverse not found in my collection list");
 
         // Superuser gets the list of Dataverses/Collections it has access to
         retrieveMyCollectionListResponse = UtilIT.retrieveMyCollectionList(superUserApiToken, null);

--- a/src/test/java/edu/harvard/iq/dataverse/api/DataverseFeaturedItemsIT.java
+++ b/src/test/java/edu/harvard/iq/dataverse/api/DataverseFeaturedItemsIT.java
@@ -89,7 +89,7 @@ public class DataverseFeaturedItemsIT {
     }
 
     @Test
-    public void testCreateFeaturedItemWithBadDvOdbjectIds() {
+    public void testCreateFeaturedItemWithBadDvObjectIds() {
         // Set up a new published dataverse and dataset
         String apiToken = createUserAndGetApiToken();
         String dataverseAlias = createDataverseAndGetAlias(apiToken);
@@ -100,7 +100,7 @@ public class DataverseFeaturedItemsIT {
         UtilIT.publishDatasetViaNativeApi(datasetId, "major", apiToken).prettyPrint();
         String datasetPersistentIdBad = datasetPersistentId + "BAD";
         String dataverseAliasBad = dataverseAlias + "BAD";
-        String fieldIdBad = "999";
+        String fieldIdBad = String.valueOf(Integer.MAX_VALUE);
 
         // Test with bad Dataset id should return bad request with not found message
         Response createFeatureItemResponse = UtilIT.createDataverseFeaturedItem(dataverseAlias, apiToken, null, 0, null, "dataset", datasetPersistentIdBad);

